query.py

```diff
@@ -24,7 +24,7 @@ def getFromMega(config):
 
     # extensions filter
     extracted = [file for file in extracted if file.endswith(tuple(ext for ext in config[FILE_TYPES]))]
-    
+
     while True:
```

Comment:
we should put the check of duplicates in this loop right after the size check i cant see where it currently exists but it should go here

maybe pull out the verification into a helper called validate_image or something like that to make the loop purpose clearer